ethereum/eip712-v2.schema.json

```
@@ -0,0 +1,103 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "properties": {
+        "blockchainName": {
+            "enum": [
+                "ethereum"
+            ]
+        },
+        "chainId": {
+            "enum": [
+                1
+            ]
+        },
+        "contracts": {
+            "items": {
+                "properties": {
+                    "address": {
+                        "pattern": "^0x[a-z0-9]{40}$",
+                        "type": "string"
+                    },
+                    "contractName": {
+                        "type": "string"
+                    },
+                    "messages": {
+                        "items": {
+                            "additionalProperties": false,
+                            "properties": {
+                                "mapper": {
+                                    "additionalProperties": false,
+                                    "properties": {
+                                        "fields": {
+                                            "items": {
+                                                "additionalProperties": false,
+                                                "properties": {
+                                                    "assetPath": {
+                                                        "type": "string"
+                                                    },
+                                                    "format": {
+                                                        "enum": [
+                                                            "coin",
```

I suggest we follow the nomenclature defined on the nano app for clarity: https://github.com/LedgerHQ/app-ethereum/blob/feat/apa/eip712_improvements/doc/ethapp.adoc#eip712-filtering
```suggestion
                                                            "token",
```

---

sure